text/0016-named-tracers.md

```diff
@@ -14,41 +14,49 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 
 The mechanism of "Named Tracers and Meters" proposed here is motivated by following scenarios:
 
-* For a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+**Expensive instrumentation**
 
-* If a library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with those instrumentation libraries later.
+For an operator of OpenTelemetry, there is currently no possibility of influencing the amount of the data produced by reporting libraries. Reporting libraries can easily "spam" backend systems, deliver bogus data, or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
```

Comment:
```suggestion
For an operator of an application using OpenTelemetry, there is currently no way to influence the amount of data produced by reporting libraries. Reporting libraries can easily "spam" backend systems, deliver bogus data, or -- in the worst case -- crash or slow down applications. These problems might even occur suddenly in production environments because of external factors such as increasing load or unexpected input data.
```